pkg/exchange/okex/okexapi/get_order_histories_request.go

```diff
@@ -0,0 +1,40 @@
+package okexapi
+
+import (
+	"time"
+
+	"github.com/c9s/requestgen"
+)
+
+//go:generate GetRequest -url "/api/v5/trade/orders-history-archive" -type GetOrderHistoriesRequest -responseDataType .APIResponse
+type GetOrderHistoriesRequest struct {
```

Comment:
just noticed that "history" is uncountable, so it should be "OrderHistory" not "OrderHistories"